tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/PermissionService.java

```diff
@@ -22,6 +22,7 @@ public class PermissionService {
   protected static final String VIEW_SENSITIVE_DATA_ETL_ROLE = "ETL";
   protected static final String VIEW_SENSITIVE_DATA_ROLE = "personsensitive:view:entities";
   protected static final String EDIT_SENSITIVE_DATA_ROLE = "personsensitive:add:modify:entities";
+  protected static final String DELETE_POST_DATA_ROLE = "post:delete:entities"; // should it be "post:delete:posts"?
```

Comment:
I don't quite understand what the relationship between the 2 permissions: `post:delete:entities` and `tcs:delete:entities`

Reply:
I have changed it